docs/content/concepts/logging/loggers.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: Loggers | Dagster
+title: Custom loggers | Dagster
 description: Dagster includes a rich and extensible logging system.
 ---
 
-# Loggers
+# Custom loggers
```

Comment:
A lot of the content in this page seems relevant to the built in logger as well as custom ones. what was the reasoning behind changing the title?

Reply:
This was an ask from Yuhan - it's possible I missed some context in Notion somewhere or misunderstood the ask. @yuhan, could you weigh in please?

Reply:
[this part](https://docs.dagster.io/concepts/logging/loggers#using-loggers) is still relevant to built-in logger, while other parts on this page talk about "custom loggers". the motivation of my ask ([here](https://www.notion.so/dagster/Essentials-v1-Plan-39a3d2a86f5f45b7bd45df8ccc658e43?pvs=4#75b741fb139849e7801f08717d1c2a32)) was to move the basic logging use case to essentials (because we believe that is an essential need of a dagster user) and leave the customization in advanced. 

concretely, the content in "using loggers" ([here](https://docs.dagster.io/concepts/logging/loggers#using-loggers) - basically the usage of `context.log`) can be in "Concepts > Logging", i.e. non-advanced. everything else will remain in the Advanced section.